attribute_set_completeness/models/attribute_set_owner_mixin.py

```diff
@@ -19,7 +19,7 @@ class AttributeSetOwnerMixin(models.AbstractModel):
         related="attribute_set_id.attribute_set_completeness_ids", readonly=True,
     )
     attribute_set_not_completed_ids = fields.Many2many(
-        comodel_name="attribute.set.completeness", compute="_compute_completion_rate",
+        comodel_name="attribute.set.completeness", readonly=True
```

Comment:
Why changing the field to a non computed one ?

The here under _compute_completion_rate method is still filling it...